src/comments.js

```diff
@@ -82,7 +82,7 @@ function decorateComment(node, comment, text) {
 
     if (
       locStart(child) - locStart(comment) <= 0 &&
-        locEnd(comment) - locEnd(child) <= 0
```

Comment:
Can you revert all the changes you've triggered by running prettier on that file? :)